platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py

```diff
@@ -448,25 +448,12 @@ def get_change_event(self, timeout=0):
             timeout = MAX_SELECT_DELAY
             while True:
                 status = self.sfp_event.check_sfp_status(port_dict, timeout)
-                if not port_dict == {}:
+                if not bool(port_dict):
                     break
```

Comment:
You want to break here if the dictionary is *not* empty, correct? If so, this should read:

```python
                if bool(port_dict):
                    break
```

Reply:
you are absolutely correct.